### DISCUSSION TITLE
Examples of mixed datasets? #DataWorkingGroup

### DISCUSSION THREAD

**Discussion by IntegratorBrad** [Project-MONAI/MONAI#1820] [Q&A]
Hello - a topic that came up recently in some of the working group meetings is getting a sense from the community are examples of mixed datasets that we can model. For example, what datasets have been seen that include both medical images as well as a piece of discrete clinical data?

As we think about how these types of data could be linked together (for example, see what the Data WG modelled last year at https://github.com/Project-MONAI/MONAI/wiki/Exploring-FHIR#example-coding-endoscope-device-output using a FHIR bundle of bundles describing encoding of related anesthesiology machine measurements) - it would be great to get some real world examples.

**Comment by pdogra89** [comment]
Mixed datasets would be very relevant for MONAI. Like we discussed, let's start with TCIA COVID-19 dataset 
https://covid-segmentation.grand-challenge.org/Data/
This would be an experiment for FHIR usability just as a proof point for being ingested in MONAI pipeline.

@Nic-Ma @IntegratorBrad

**Reply by pdogra89** [reply]
@thewtex - It would be great to see how ITK toolkit can read the tags from same dataset (DICOM) in this case 
 https://covid-segmentation.grand-challenge.org/Data/ as a proof point to pass the structure/tags to MONAI for ingestion.

**Reply by pdogra89** [reply]
@IntegratorBrad @wyli  - On second thoughts, wondering why we go FHIR route ? the goal is mixed datasets ingestion for MONAI.
1) Read the image & metadata from DICOM - https://covid-segmentation.grand-challenge.org/Data/
2) Read image & metadata in tabular form (csv) - do we have such a dataset ? if not can we just create a dummy example for data ingestion example ?

@IntegratorBrad - I didn't follow the need for FHIR as first prototype goal from Data working group ?
